pkg/ocmagenthandler/ocmagenthandler_deployment.go

```diff
@@ -215,15 +215,21 @@ func buildOCMAgentArgs(ocmAgent ocmagentv1alpha1.OcmAgent) []string {
 		fmt.Sprintf("--access-token=@%s", accessTokenPath),
```

Comment:
We will need to move `--access-token` along with `--cluster-id` flag in non-fleet-mode